### DISCUSSION TITLE
API support

### DISCUSSION THREAD

**Discussion by sertaco** [Bing-su/adetailer#75] [Ideas]
Hi guys,
I need API support for this plugin. It does wonders but for productionizing this API support is a must have. 

Are there any pointers that I can follow for running this with automatic1111's fast api?

**Comment by Bing-su** [comment]
Refer to https://github.com/mix1009/sdwebuiapi and my [wiki](https://github.com/Bing-su/adetailer/wiki/API), send a request to `/sdapi/v1/txt2img` or `/sdapi/v1/img2img` with `After Detailer` in alwayson_scripts.

**Reply by sertaco** [reply]
Thanks @Bing-su. Looks very promising. 

How about installing the extension programmatically? I need to bake this into a dockerfile so installing with the UI is not an option unfortunately.

**Reply by sertaco** [reply]
My understanding is that this does only git clone this repo into extensions folder and that is it. Can you confirm?

**Reply by Bing-su** [reply]
Yes, that's it.
